about.html

```diff
@@ -12,52 +12,73 @@
 </head>
 
 <body>
-    <div class="container about-page">
+  <header>
+    <div class="lambda-logo">
+    <img src="\img\lambda-black.png " alt="lambda-logo"/>
+    </div>
+    <div class="header nav">
+    <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+    </nav>
+    </div>
+    </header>
+
+<div class="container-about">
+        
+   
+  <section class="top-content">
+      <img src="\img\jumbo-about.png" alt="header-image"/>
+      <div class="text-container">
      
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
+    <h2>About</h2>
```

Comment:
You are making good progress with this.  This is the tricky part of the pages.  You can further help with targeting CSS by surrounding each of the 4 sections as well.  One thing that really helps here is remembering that flexbox can only go 2 levels deep, and in this situation, it really helps you format this page properly.